lib/services/inventory/add_inventory_audit.dart

```diff
@@ -6,23 +6,23 @@ import '../../app_config.dart';
 import '../storage_service.dart';
 
 class AddInventoryAudit {
-  static final apiKey = null;
   Post _post = Post();
-  Future<AddInventoryAuditModel> login(String auditCommentType,
-      String auditMessage, int doRefreshToken, int inventoryId) async {
+  Future<AddInventoryAuditModel> addinventory(
+      String auditCommentType, String auditMessage, String inventoryId) async {
     final prodUrl = await AppConfig.forEnvironment('prod');
-    final loginUrl = prodUrl.baseUrl + "/attendance/API_HR/api.php";
-
+    final auditUrl = prodUrl.baseUrl + "/attendance/API_HR/api.php";
```

Comment:
this looks wrong. such url should come from on single server

Reply:
changed it